.gitignore

```diff
@@ -1,5 +1,5 @@
 HELP.md
-src/main/resources/application.properties
+*.properties
```

Comment:
properties files should be in the repository
if you have sensitive data in there - that should be encrypted or replaces with placeholders